### PR TITLE
Add landing message. #1

### PR DESCRIPTION
**Pull request by harry-p0tt3r**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1 +1,12 @@
...
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE-edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Welcome to Potionfy</title>
+    </head>
+    <body>
+        <h1>Welcome to Potionfy! Enjoy your stay.</h1>
```
Comment (HermioneDeBcc) [inline index.html]: Could you remove the `Enjoy your stay.` string and make the `Welcome to Potionfy!` string red?

Comment (harry-p0tt3r) [reply index.html]: Sure, can do!

**Review by HermioneDeBcc — CHANGES_REQUESTED** [review]
Please,change the code as per the comment in my review.

**Review by HermioneDeBcc — APPROVED** [review]
Well done Harry!

**Pull request merged**